doc/user-guides/acrn-shell.rst

```diff
@@ -50,3 +50,179 @@ The ACRN hypervisor shell supports the following commands:
    * - wrmsr [-p<pcpu_id>] <msr_index> <value>
      - Write ``value`` (in hexadecimal) to the Model-Specific Register (MSR) at
        index ``msr_index`` (in hexadecimal) for CPU ID ``pcpu_id``
+
+Command examples
+****************
+
+The following sections provide further details and examples for some of these commands.
+
+vm_list
+=======
+
+"vm_list" provides the name of each virtual machine and its corresponding ID and
```

Comment:
Wouldn't it be better to use double-backticks to enclose these commands? I.e.:
```suggestion
``vm_list`` provides the name of each virtual machine and its corresponding ID and
```

(same question for all the other ones below)

Reply:
david, can you help me to replace these double-backticks and if you can give me a google doc, let me to modify it, thanks.